### DISCUSSION TITLE
Login error all the sudden?

### DISCUSSION THREAD

**Discussion by aries223** [pfelk/pfelk#434] [Q&A]
I am not sure why.. but I cannot log in to the Elastic dashboard anymore. I just get "_We couldn't log you in. Please try again._". 

I have tried re-running this:
sudo /usr/share/elasticsearch/bin/elasticsearch-reset-password -u elastic
but it errors out: "_ERROR: Failed to reset password for the [elastic] user_"

How do I reset this / get past this error?

**Comment by a3ilson** [comment]
@aries223 can you clarify? 

Were you able to login before? If so what occurred prior to this issue? 
Did you always login with the built-in 'elastic' user? It is recommended that you login with the 'pfelk_viewer' credentials.

Have you tried terminating Kibana and logstash prior to resetting the elastic user credentials?

**Comment by aries223** [comment]
Yes I was always able to log in, now none of the logins work. I have not made any configuration changes beyond the initial setup directions (except for configuring dashboards). 
Yes I tried terminating Kibana and logstash prior to resetting the elastic user credentials, same error.

The only changes would be system updates, I believe recently there was also an Elastic update as well.

Is there a config file, or a command I can run to reset/change the login user password?

P.S. running this (per Elastic website) to get the version number gives me a login error:

```
curl --cacert $ES_PATH_CONF/certs/http_ca.crt -u elastic https://192.168.1.154:9200
Enter host password for user 'elastic':
curl: (7) Failed to connect to 192.168.1.154 port 9200: Connection refused
```

**Comment by a3ilson** [comment]
You'll need to specify the `http_ca.crt` location 
```
curl --cacert /etc/elasticsearch/certs/http_ca.crt -u elastic https://192.168.1.154:9200
Enter host password for user 'elastic':
curl: (7) Failed to connect to 192.168.1.154 port 9200: Connection refused
```

**Reply by aries223** [reply]
curl: (77) error setting certificate verify locations:
  CAfile: /etc/elasticsearch/certs/http_ca.crt
  CApath: /etc/ssl/certs

**Reply by aries223** [reply]
...

**Comment by aries223** [comment]
...

**Reply by a3ilson** [reply]
I would look at the log files for both Kibana and elastic...something is causing them to crash.
